src/clients/discord.ts

```diff
@@ -0,0 +1,125 @@
+import { AnyChannel, Client, Collection, Message, PartialMessage, TextChannel } from 'discord.js';
+import { linkedListNode } from '../linkedList';
+import { conjoinedMsg } from '../messageObjects';
+import bridge, { configFile, genericPromiseError, manageMsgCache, twitchDelete } from './bridge';
+const { messageLinkdListInterface, twitchMessageSearchCache, discordTwitchCacheMap, lastUserStateMsg } = bridge;
```

Comment:
The default object we're importing is an object, so from my knowledge we can't just destructor it in the import. If we make it separate vars etc then we can't change them because imports are constants. Sorry if this explanation is bad on my phone:p

Reply:
This is one of those moments where I'd like to know typescript in general more XP

Kindof seems like an oof that it can't be done that way... With the old `require` function, one was able to grab module.epxorts as a standard js objet. So what confuses me I guess is the way it's being treated in TS... worth experimenting with, but it sounds like it is what it is

Reply:
yeet script

Reply:
Removed the destructoring. We are now just directly accessing `bridge.property` for everything, looks cleaner. 100% wasn't your idea. Closing cause kind of solved?